lib/versioning/semver-coerced/index.ts

```diff
@@ -11,27 +11,27 @@ export const supportsRanges = false;
 const { is: isStable } = stable;
 
 function sortVersions(a: string, b: string): number {
-  return semver.compare(semver.coerce(a), semver.coerce(b));
+  return semver.compare(semver.coerce(a) ?? a, semver.coerce(b) ?? a);
```

Comment:
Wrong, return 0 if version can't coerced

Reply:
semver.compare only accepts the type `string | semver.SemVer` if we use a 0 there it won't pass the null check.